### PR TITLE
Fix log topics

### PR DESCRIPTION
**Pull request by benbierens**
It's giving a lot of spam in Codex.
With log topics, we can filter.

### CODE REVIEW

File: serde/json/deserializer.nim [hunk 1]
```diff
@@ -188,6 +185,9 @@ proc fromJson*[T: ref object or object](_: type T, json: JsonNode): ?!T =
     let isOptionalValue = typeof(value) is Option
     var skip = false # workaround for 'continue' not supported in a 'fields' loop
 
+    logScope:
+      topics = "serde json deserialization"
+
```
Comment (emizzle) [inline serde/json/deserializer.nim:190]: Why did you move it so far down?

Comment (benbierens) [reply serde/json/deserializer.nim:190]: The log topics were ignored completely when they were placed at the top of the file.
I suspect this proc is being called in a marco (or similar magical structure) somewhere, which makes chronicles not see the log topics when they're at the top of the file.
It works when I put them here.

Comment (emizzle) [reply serde/json/deserializer.nim:190]: Oh wow, ok. I've noticed this in the past within codex, but had not figured out a fix. That really sucks. We should open an issue in chronicles as well.

Would you mind adding a comment to that effect, please?

Comment (benbierens) [reply serde/json/deserializer.nim:190]: Issue filed here: https://github.com/status-im/nim-chronicles/issues/148

**Review by emizzle — APPROVED** [review]
LGTM, the chronicles bug is unfortunate, but we can update once it's addressed.
